lockfile/src/parsers/go_sum.rs

```diff
@@ -0,0 +1,51 @@
+use phylum_types::types::package::{PackageDescriptor, PackageType};
+
+use super::*;
```

Comment:
I really don't like this habit in our parsers and it has cost me some completely unnecessary debugging time in the past. So I'd recommend to import everything directly in this file for the nom stuff we need since it's a new file anyway.

It's not like we need to bundle imports in one location to reduce file size, this is just unnecessary indirection that makes troubleshooting and adding additional imports more complicated.

Reply:
Makes sense, done